### PR TITLE
DOCS-#4027, #4211: Add GIF images and chart to Modin README demonstrating speedups

### LINKED ISSUE #4027: Add a GIF to the Modin README demonstrating speedups
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. What kind of performance improvements would you like to see with this new API?

`From @dorisjlee in #3929 : I think it is important for us to think about how we could add a quick demo here that demonstrates how Modin speeds up pandas right out of the box. People who are looking at the README are likely people who are considering whether or not to they should try out Modin. Anything we can do here to shorten the time to value for potential users would be helpful for adoption.

For reference, I tried coming up with an example like this on the 10-min quickstart guide. Even that small example takes a while to run because of the time to download the dataset. We can reuse the same example, but maybe swap with a smaller dataset (so it's faster to download) and duplicate it many more times to generate the big dataframe. Or we can generate synthetic data on the fly like in our current README example. Then followed by a demo of one or two operators comparing Modin and pandas performance. Right now the README example only shows we can create a big dataframe but provides no reference comparison on why this is useful.

Couple ideas that might help with the presentation for this killer demo/example. Maybe in addition to the code, make a GIF that shows pandas v.s. Modin being executed side by side, we can speed up the GIF playback speed to illustrate our point. [pandarallel]'s README has a good example of this. The most effective GIFs are ones where it is short and highlights the strongest value prop for the tool (e.g., in Lux, our GIF showed click on Toggle button and we turn this pandas table into pretty charts for free!)

We can play around with a combination of code+GIF or GIF-only, depending on whether we think the code is too long and hard to parse at a glance. We can also add a link to try out the example on a live notebook as suggested in #3897.`

### LINKED ISSUE #4211: DOCS: Remove some tutorials from modin/examples
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **Modin version** (`modin.__version__`):
- **Python version**:
- **Code we can use to reproduce**:

### Describe the problem

We should remove both of these tutorials from `modin/examples` https://github.com/modin-project/modin/tree/master/examples/jupyter, and they are already covered by https://github.com/modin-project/modin/blob/master/examples/tutorial/tutorial_notebooks/introduction/exercise_2.ipynb and the links as well as the `read_csv` calls are outdated.

### PR DESCRIPTION
**Pull request by naren-ponder**
Signed-off-by: Naren Krishna <naren@ponder.io>

## What do these changes do?

Add a GIF comparing Modin vs. Pandas execution on Taxi dataset to the Modin README. Related to #4027.

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/development/contributing.html#commit-message-formatting)
- [x] passes `flake8 modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] passes `black --check modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] signed commit with `git commit -s` 
- [x] Resolves #4027 ? 
- [x] Resolves #4211  ? 
- [x] tests added and passing
- [x] module layout described at `docs/development/architecture.rst` is up-to-date 
- [x] added (Issue Number: PR title (PR Number)) and github username to release notes for next major release

### CODE REVIEW

**Review by mvashishtha — COMMENTED** [review]
Thanks for taking the time to make these.

- Will you insert these images and gifs into the docs in this PR?
- The text in the gifs looks really small for me. Should we capture them in a notebook with a higher font size? Should we zoom into the notebook?
- Should the gifs be narrower?

**Review by jeffreykennethli — COMMENTED** [review]
For `Modin_Speedup.png`, I think the function names would look better with some combination of: not bolded, another font, in a box, with a title separator, etc. The current look makes it hard to set apart.

The gifs look like they have very low resolution - will that affect the final rendering on the README?

**Comment by dorisjlee** [comment]
Thanks @naren-ponder ! I agree with @mvashishtha the text on the GIF is a bit small. Increasing the resolution and fontsize would be better. Note that right now the `parse_date` and `quoting` on read_csv is what's making the cell really wide. If we can remove those, then you would be able to record the video at a much greater zoom level on the notebook. Also agree with @jeffreykennethli comments about the hard-to-read font sizes on the PNG.

**Review by jeffreykennethli — COMMENTED** [review]
`Modin_Speedup.png` text is inconsistent - applymap() says "faster" instead of speedup. On a side note, faster sounds better to me as long as it's consistent of course.

**Review by devin-petersohn — DISMISSED** [review]
LGTM, thanks @naren-ponder

File: README.md [hunk 1]
```diff
@@ -18,12 +18,30 @@ single-threaded, Modin lets you instantly speed up your workflows by scaling pan
...
-
-```python
-# import pandas as pd
-import modin.pandas as pd
-```
+By simply replacing the import statement, Modin offers users effortless speed and scale for their pandas workflows:
+
+<img src="docs/img/Modin_Pandas_Import.gif" style="display: block;margin-left: auto;margin-right: auto;" width="100%"></img>
+
+In the GIFs below, Modin (left) and pandas (right) *perform the same pandas* operations on a 2GB dataset. The only difference between the two notebook examples is that Modin uses `import modin.pandas as pd` and pandas uses `import pandas as pd`.
```
Comment (mvashishtha) [inline README.md:25]:
```suggestion
In the GIFs below, Modin (left) and pandas (right) perform *the same pandas operations* on a 2GB dataset. The only difference between the two notebook examples is that Modin uses `import modin.pandas as pd` and pandas uses `import pandas as pd`. 
```

File: README.md [hunk 2]
```diff
@@ -18,12 +18,30 @@ single-threaded, Modin lets you instantly speed up your workflows by scaling pan
...
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-0lax"><img src="docs/img/Modin.gif"></img></td>
+    <td class="tg-0lax"><img src="docs/img/Pandas.gif"></img></td>
+  </tr>
+</tbody>
+</table>
+
+The chart below summarizes the runtime speedup of replacing Modin with Pandas based on the examples above. The example notebooks can be located [here](examples/jupyter). To learn more about the speedups you could get with Modin, check out our [10 minute quickstart guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html) to try out some examples on your own!
```
Comment (mvashishtha) [inline README.md:42]:
```suggestion
The chart below summarizes the runtime speedup of replacing Modin with Pandas based on the examples above. The example notebooks can be found [here](examples/jupyter). To learn more about the speedups you could get with Modin and try out some examples on your own, check out our [10 minute quickstart guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html)!
```

File: examples/jupyter/Modin_Taxi.ipynb [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+   "execution_count": 1,
+   "id": "cc4bd9e9",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "skip"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# To run this notebook as done in the README GIFs, you must first locally download the 2015 NYC Taxi Trip Data.\n",
```
Comment (mvashishtha) [inline examples/jupyter/Modin_Taxi.ipynb:14]: Why comment this part out? The notebook can't be run without these lines.

File: examples/jupyter/Pandas_Taxi.ipynb [hunk 4]
```diff
@@ -0,0 +1,116 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5d674ce8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# To run this notebook as done in the README GIFs, you must first locally download the 2015 NYC Taxi Trip Data.\n",
```
Comment (mvashishtha) [inline examples/jupyter/Pandas_Taxi.ipynb:10]: Ditto: Why comment out the download lines?

**Review by mvashishtha — DISMISSED** [review]
LGTM! Thanks @naren-ponder

**Review by jeffreykennethli — APPROVED** [review]
nit: in `docs/img/Modin_Speedup.svg` the 2.5, 4.1, 5.1X faster text look better with lowercase "x" in my opinion.

2.5x faster! 4.1x faster! 5.1x faster!

otherwise, LGTM ty for the change!

File: README.md [hunk 5]
```diff
@@ -18,12 +18,30 @@ single-threaded, Modin lets you instantly speed up your workflows by scaling pan
...
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-0lax"><img src="docs/img/Modin.gif"></img></td>
+    <td class="tg-0lax"><img src="docs/img/Pandas.gif"></img></td>
+  </tr>
+</tbody>
+</table>
+
+The chart below summarizes the runtime speedup of replacing Modin with Pandas based on the examples above. The example notebooks can be found [here](examples/jupyter). To learn more about the speedups you could get with Modin and try out some examples on your own, check out our [10 minute quickstart guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html) to try out some examples on your own!
```
Comment (dorisjlee) [inline README.md:42]:
```suggestion
The charts below show the speedup you get by replacing pandas with Modin based on the examples above. The example notebooks can be found [here](examples/jupyter). To learn more about the speedups you could get with Modin and try out some examples on your own, check out our [10-minute quickstart guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html) to try out some examples on your own!
```

File: README.md [hunk 6]
```diff
@@ -18,12 +18,30 @@ single-threaded, Modin lets you instantly speed up your workflows by scaling pan
...
+<thead>
+  <tr>
+    <th class="tg-0lax" style="text-align: center;"><img src="https://github.com/modin-project/modin/blob/master/docs/img/MODIN_ver2_hrz.png?raw=True" height="35px"></th>
+    <th class="tg-0lax" style="text-align: center;"><img src="https://pandas.pydata.org/static/img/pandas.svg" height="50px"></img></th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-0lax"><img src="docs/img/Modin.gif"></img></td>
+    <td class="tg-0lax"><img src="docs/img/Pandas.gif"></img></td>
```
Comment (dorisjlee) [inline README.md:37]: For both the links to `examples/jupyter` and GIFs, is it better to use absolute links to the URL or local refs? My sense is that absolute links are better so that README still work in isolated contexts. If so, let's replace the links, which means the links will break for now until the PR is merged.

File: README.md [hunk 7]
```diff
@@ -18,12 +18,30 @@ single-threaded, Modin lets you instantly speed up your workflows by scaling pan
...
+  </tr>
+</tbody>
+</table>
+
+The chart below summarizes the runtime speedup of replacing Modin with Pandas based on the examples above. The example notebooks can be found [here](examples/jupyter). To learn more about the speedups you could get with Modin and try out some examples on your own, check out our [10 minute quickstart guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html) to try out some examples on your own!
+
+<img src="docs/img/Modin_Speedup.svg" style="display: block;margin-left: auto;margin-right: auto;" width="100%"></img>
```
Comment (dorisjlee) [inline README.md:44]: change to absolute links?

**Review by dorisjlee — CHANGES_REQUESTED** [review]

**Review by devin-petersohn — APPROVED** [review]

File: docs/getting_started/examples.rst [hunk 8]
```diff
@@ -10,8 +10,7 @@ Usage Examples
 The following notebooks demonstrate how Modin can be used for scalable data science:
 
 - Quickstart Guide to Modin [`Source <https://github.com/modin-project/modin/tree/master/examples/quickstart.ipynb>`_]
-- Using Modin with the NYC Taxi Dataset [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi.ipynb>`_]
-- Using Experimental Modin in the cloud with the NYC Taxi Dataset on a AWS cluster [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi_cloud.ipynb>`_]
```
Comment (YarShev) [inline docs/getting_started/examples.rst:14]: This should be removed in https://modin.readthedocs.io/en/stable/usage_guide/examples/index.html#modin-in-the-cloud too.

**Review by YarShev — COMMENTED** [review]
A general comment to new added files. Why do those contain capital letters? I think we should have lower-case names for files.
